doc/sphinx/administration/configuration/osh-http-proxy_conf.rst

```diff
@@ -0,0 +1,184 @@
+=============================
+osh-http-proxy.conf reference
+=============================
+
+.. note::
+
+   This module is optional, and disabled by default. To know more about the HTTP Proxy feature
+   of The Bastion, please check :doc:`/using/httpproxy`
+
+Option List
+===========
+
+
+HTTP Proxy configuration
+------------------------
+
+These options modify the behavior of the HTTP Proxy, an optional module of The Bastion
+
+- :ref:`enabled`
+- :ref:`port`
+- :ref:`ssl_certificate`
+- :ref:`ssl_key`
+- :ref:`ciphers`
+- :ref:`insecure`
+- :ref:`min_servers`
+- :ref:`max_servers`
+- :ref:`min_spare_servers`
+- :ref:`max_spare_servers`
+- :ref:`timeout`
+- :ref:`log_request_response`
+- :ref:`log_request_response_max_size`
+
+Option Reference
+================
+
+HTTP Proxy configuration
+------------------------
+
+.. _enabled:
+
+enabled
+*******
+
+:Type: ``bool``
+
+:Default: ``false``
+
+Whether the HTTP proxy daemon daemon is enabled or not. If it's not enabled, it'll exit when started. Of course, if you want to enable this daemon, you should **also** configure your init system to start it for you. Both sysV-style scripts and systemd unit files are provided. For systemd, using `systemctl enable osh-http-proxy.service` should be enough. For sysV-style inits, it depends on the scripts provided for your distro, but usually `update-rc.d osh-http-proxy defaults` then `update-rc.d osh-http-proxy enable` should do the trick.
+
+.. _port:
+
+port
+****
+
+:Type: ``int, 1 to 65535``
+
+:Default: ``8443``
+
+The port to listen to. You can use ports < 1024, in which case privileges will be dropped after binding, but please ensure your systemd unit file starts the daemon as root in that case.
+
+.. _ssl_certificate:
+
+ssl_certificate
+***************
+
+:Type: ``string``
+
+:Default: ``/etc/ssl/private/ssl-cert-snakeoil.key``
```

Comment:
```suggestion
:Default: ``/etc/ssl/certs/ssl-cert-snakeoil.pem``
```

Reply:
This doc is generated, fixed it in the source file instead !